style.css

```diff
@@ -0,0 +1,77 @@
+body {
+  text-align: center;
+  color: white;
+}
+
+#main {
+  background-color: #94c473;
+  display: flex;
+  flex-direction: column;
+  padding: 60px 80px;
+  flex: 1 1 0%;
+}

```

Comment:
I think it would be better if you use Css Grid on the main container and flex for the childs 
that way you have a better-structured layout 
in this example, the image layout could be much better !